contracts/Market.sol

```diff
@@ -356,7 +358,44 @@ contract Market is IMarket, Ownable, ERC721 {
 
 		emit Settled(index, _bets[index].payout, result, recipient);
 
-		_burn(index);
+		_burn(uint256(index));
+	}
+
+	/* 
+	 * @dev Reverse a bet. Return the stake to the bettor and the loan to the vault
+	 * @param index The index of the bet
+	 * @param signature Signature from market owner
+	 */
+	function refund(uint64 index, SignatureLib.Signature calldata signature) external {
+		bytes16 REFUND_CMD = 0x726566756e6400000000000000000000; //Bytes for "refund"
+		bytes32 messageHash = keccak256(
+			abi.encodePacked(REFUND_CMD, address(this), index)
```

Comment:
not sure what the point of passing `REFUND_CMD` is - wouldnt the `signature` contain the index of the bet that is to be refunded, and then the sig validity is checked?

Reply:
> Using an instruction string and contract address in signed message. Instruction string refund makes the intent of the signed message clear. 

If you're signing something that is just a number, it's not clear what the intent of the signature is. Including a parameter that provides the intent means that the signatory can gain some clarity on what the signature is for.